packages/cozy-dataproxy-lib/src/components/Search/useFetchResult.jsx

```diff
@@ -7,6 +7,31 @@ import { getIconForSearchResult } from './getIconForSearchResult'
 
 const log = Minilog('🔍 [useFetchResult]')
 
+const searchWithRetry = async (
+  dataProxy,
+  searchValue,
+  { maxRetries = 5, delay = 500 } = {}
+) => {
+  let currentDelay = delay
+  // Make several search attemps in case it is not ready yet
+  for (let attempt = 0; attempt < maxRetries; attempt++) {
+    const searchResults = await dataProxy.search(searchValue)
+
+    if (searchResults && Array.isArray(searchResults)) {
```

Comment:
If the search is unsuccesful, what can it return ? null ? Or something else ?

Reply:
If the index are not initialized, null will be returned
If the index *are* initialized, and the search is working it will be an array

Reply:
So theoretically no need to check `Array.isArray(searchResults)` ? But it's a nit, no problem.

Reply:
Yes, I guess we could just do `if (searchResults)` and avoid an additional check :+1:

Reply:
Fixed within 344c30cb2ba34dd7bd5b9cbcd37e312dbeff1848